app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
```

Comment:
```suggestion
  def full_title page_title
```
(tat) làm rails tut sửa rồi giờ vẫn mắc